### PR TITLE
fix: added global styles for checkbox focus state

### LINKED ISSUE #2149: Bug: Checkboxes on the User Settings Page Need Focus States
### Describe the bug

Checkboxes on the User Settings Page Need Focus States

### Steps to reproduce

1. Go to your user settings page, https://app.opensauced.pizza/user/settings
2. Use the <kbd>TAB</kbd> key to tab through all the fields. Notice that once you hit a checkbox, you can change it's state with the keyboard, but it never receives a focus state.

CleanShot 2023-11-14 at 22 41 55

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Now checkboxes have focus state styling.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2149

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2023-11-27 at 11 49 27

**After**

CleanShot 2023-11-27 at 11 48 27

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -25,6 +25,14 @@ h6 {
   @apply tracking-tight;
 }
 
+input[type="checkbox"]:focus-visible,
+[role="checkbox"]:focus-visible {
```
Comment (nickytonline) [inline styles/globals.css:29]: The [Radix checkbox](https://www.radix-ui.com/primitives/docs/components/checkbox) is a button with a role of checkbox.

**Comment by isabensusan** [comment]
Thanks @nickytonline !! This looks so much better already.

The one thing we are missing that I really like on the checkbox and would love to see on every other input is the focus ring:
(the lighter orange ring on the outside that is displayed when the user focuses on the checkbox)
image

Here's the css for the focus style on checkboxes:
image

(and here's the copy/paste friendly version)
```
[role=checkbox]:focus-visible, [role=checkbox]:hover, input[type=checkbox]:focus-visible {
    outline: 2px solid transparent;
    outline-offset: 2px;
    --tw-ring-offset-shadow: var(--tw-ring-inset) 0 0 0 var(--tw-ring-offset-width) var(--tw-ring-offset-color);
    --tw-ring-shadow: var(--tw-ring-inset) 0 0 0 calc(3px + var(--tw-ring-offset-width)) var(--tw-ring-color);
    box-shadow: var(--tw-ring-offset-shadow),var(--tw-ring-shadow),var(--tw-shadow,0 0 #0000);
    --tw-border-opacity: 1;
    border-color: rgb(249 115 22/var(--tw-border-opacity));
    --tw-ring-opacity: 1;
    --tw-ring-color: rgb(255 237 213/var(--tw-ring-opacity));
}
```



On a separate note -- I'd love love love if we could change the look on our checkboxes just a tiny bit to make it feel more modern:
Before
image

After
image

I achieved the look fairly easily by changing the checkbox's svg stroke-width to 5 and adding the class `.p-0.5` to the span wrapping the svg.

**Review by NsdHSO — APPROVED** [review]
LGTM

**Review by jpmcb — APPROVED** [review]
✅

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**